### PR TITLE
Updated login for mozillians tests

### PR DESCRIPTION
**Pull request by viorelaioia**

### CODE REVIEW

**Comment by viorelaioia** [comment]
@davehunt , @johngian , can you please take a look over this PR? Thanks!

File: pages/authentification.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import email
+
+import time
+from tempmail import TempMail
+from selenium.webdriver.common.by import By
+
+from pages.base import Base
+
+
+class Authentification(Base):
```
Comment (davehunt) [inline pages/authentification.py]: I'd rename this file to be `auth_zero.py` and the class to be `AuthZero`, and I don't think it should extend from `Base` as it's a page from another party.

File: pages/authentification.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+    _login_with_email_button_locator = (By.CSS_SELECTOR, '.auth0-lock-passwordless-button.auth0-lock-passwordless-big-button')
+    _email_input = (By.CSS_SELECTOR, '.auth0-lock-passwordless-pane>div>div>input')
```
Comment (davehunt) [inline pages/authentification.py]: Missing `_locator` suffix.

File: pages/authentification.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+
+    def sign_in(self, email):
+        self.wait_for_element_visible(*self._login_with_email_button_locator)
+        self.selenium.find_element(*self._login_with_email_button_locator).click()
+        self.wait_for_element_visible(*self._email_input)
+        self.selenium.find_element(*self._email_input).send_keys(email)
+        self.selenium.find_element(*self._send_email_button_locator).click()
+        # Wait for the email to be received 5 seconds after login
+        time.sleep(5)
```
Comment (davehunt) [inline pages/authentification.py]: We should have a wait, not a sleep. In some cases 5 seconds might not be enough, and in a lot of cases it may be too much.

Comment (davehunt) [inline pages/authentification.py]: The method name `sign_in` suggests it will complete the sign-in process, however this stops after requesting the email. We should either rename the method for clarity, or complete the process here.

File: pages/authentification.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+    def get_link(self, email_address, password):
```
Comment (davehunt) [inline pages/authentification.py]: This is not related to the page, but to the email received. We should move these functions into a helper class. I would also prefer to use [restmail.net](https://github.com/mozilla/restmail.net) over tempmail as it's hosted by Mozilla. We have an example use of restmail here: https://github.com/mozilla/bidpom/blob/master/tests/restmail.py

File: pages/base.py [hunk 5]
```diff
@@ -19,6 +18,10 @@ class Base(Page):
 
     _pending_approval_locator = (By.ID, 'pending-approval')
     _account_created_successfully_locator = (By.CSS_SELECTOR, 'div.alert:nth-child(2)')
+    _login_with_email_button_locator = (By.CSS_SELECTOR, '.auth0-lock-passwordless-button.auth0-lock-passwordless-big-button')
+    _email_input = (By.CSS_SELECTOR, '.auth0-lock-passwordless-pane>div>div>input')
+    _send_email_button_locator = (By.CSS_SELECTOR, '.auth0-lock-passwordless-submit')
+    _email_sent_successful_message_locator = (By.CSS_SELECTOR, '.auth0-lock-passwordless-confirmation>p')
```
Comment (davehunt) [inline pages/base.py]: These locators should only be in the Auth0 page object.

File: pages/register.py [hunk 6]
```diff
@@ -25,6 +25,11 @@ class Register(Base):
     _recaptcha_checkbox_locator = (By.CSS_SELECTOR, '.recaptcha-checkbox-checkmark')
     _recaptcha_checkbox_checked = (By.CSS_SELECTOR, '.recaptcha-checkbox-checked')
 
+    def __init__(self, base_url, selenium, open_url=True):
+        Base.__init__(self, base_url, selenium)
+        if open_url:
+            self.selenium.get(self.base_url)
+
```
Comment (davehunt) [inline pages/register.py]: Why is this needed, where it wasn't before? This is a practice we've moved away from, which is why I ask.

File: tests/conftest.py [hunk 7]
```diff
@@ -49,3 +46,5 @@ def private_user(stored_users):
 @pytest.fixture
 def unvouched_user(stored_users):
     return stored_users['unvouched']
+
+
```
Comment (davehunt) [inline tests/conftest.py]: Nit: Remove these blank lines.

**Comment by viorelaioia** [comment]
Closing, opened https://github.com/mozilla/mozillians-tests/pull/267 instead
